### DISCUSSION TITLE
请问该库的在安全多方计算上的功能定位是怎样的?

### DISCUSSION THREAD

**Discussion by 1191000814** [secretflow/secretflow#1178] [Q&A]
和常见的安全多方计算库mpyc,mp-spdz的区别在哪(仅考虑安全多方计算), 能否实现算法复杂的安全多方计算?

**Comment by anakinxc** [comment]
Hi @1191000814 

SPU 的[论文](https://www.usenix.org/system/files/atc23-ma.pdf)应该可以回答你的问题
对于复杂的算法/模型，可以看[这里](https://github.com/secretflow/spu/tree/main/examples/python/ml)的例子

Best
~Yancheng
